pennylane/ops/qubit.py

```diff
@@ -408,27 +416,59 @@ class QubitStateVector(Operation):
     """
     num_params = 1
     num_wires = 0
-    par_domain = 'A'
-    grad_method = 'F'
-
-
-all_ops = [
-    Hadamard,
-    PauliX,
-    PauliY,
-    PauliZ,
-    CNOT,
-    CZ,
-    SWAP,
-    RX,
-    RY,
-    RZ,
-    PhaseShift,
-    Rot,
-    BasisState,
-    QubitStateVector,
-    QubitUnitary
-]
+    par_domain = "A"
+    grad_method = "F"
+
+
+# =============================================================================
+# Observables
+# =============================================================================
+
+
+class Hermitian(Observable):
+    r"""Hermitian(A, wires)
+    Observable value of an arbitrary Hermitian observable.
+
+    For a Hermitian matrix :math:`A`, this expectation command returns the value
```

Comment:
```suggestion
    For a Hermitian matrix :math:`A`, the expectation command returns the value
```

Reply:
Though this docstring probably needs an update to be better aligned with this refactor